src/migration/1759384808784-DropUserExperienceJobSkills.ts

```diff
@@ -0,0 +1,106 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class DropUserExperienceJobSkills1759384808784
+  implements MigrationInterface
+{
+  name = 'DropUserExperienceJobSkills1759384808784';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
```

Comment:
FYI, you don't need to alter table to drop constraint before you drop the table, because it gets done implicitly when dropping the table

Reply:
<img width="717" height="267" alt="Screenshot 2025-10-07 at 5 25 22 PM" src="https://github.com/user-attachments/assets/3e9bb928-88ac-4367-b040-902b0ec31e63" />

Reply:
You need to drop user_experience_skill first